code/lib/onewallet.js

```diff
@@ -222,6 +222,14 @@ const computeSetRecoveryAddressHash = ({ address }) => {
   return { hash: keccak(input), bytes: input }
 }
 
+// dest, hex string
+const computeDestHash = ({ dest }) => {
```

Comment:
To merge with computeSetRecoveryAddressHash

Reply:
done - also modified `SetRecovey.jsx` to use `computeDestHash`

Reply:
where is it changed?